public/app/features/alerting/unified/components/rules/NoRulesCTA.tsx

```diff
@@ -6,7 +6,8 @@ import { useRulesAccess } from '../../utils/accessControlHooks';
 
 const RecordingRulesButtons = () => {
   const { canCreateGrafanaRules, canCreateCloudRules } = useRulesAccess();
-  const grafanaRecordingRulesEnabled = config.featureToggles.grafanaManagedRecordingRules;
+  const grafanaRecordingRulesEnabled =
+    config.featureToggles.grafanaManagedRecordingRules && config.unifiedAlerting.recordingRulesEnabled;
```

Comment:
nit: can we use `useGrafanaManagedRecordingRulesSupport`? It looks like identical logic 🤔

Reply:
🤦 Yes, thanks